Library.Domain/Interfaces/IRepositories.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace Library.Domain.Interfaces;
+
+/// <summary>
+/// Интерфейс репозитория для CRUD-операций с объектами доменной области.
+/// </summary>
+/// <typeparam name="TEntity">Тип сущности</typeparam>
+/// <typeparam name="TKey">Тип Id сущности</typeparam>
+public interface IRepositories<TEntity, TKey>
```

Comment:
Лучше назвать в единственном числе:
```suggestion
public interface IRepository<TEntity, TKey>
```